source/manual/get-started.html.md

```diff
@@ -17,24 +17,278 @@ If you're having trouble with this guide, you can ask your colleagues or the #go
+
+Test that both tools work by running `gds --help` and `gds govuk connect --help`.
+
+> If you see `fatal: no such path in the working tree`, it's because you're using ZSH, which has `gds` set up as a Git alias. You can either remove that alias by adding `unalias gds` to your `~/.zshrc`, or use `gds-cli` instead of `gds` for all the relevant commands.
+
+The GDS CLI requires some initial configuration:
+
+```bash
+gds config email firstname.lastname@digital.cabinet-office.gov.uk
+gds config yubikey false # If you type MFA codes from your phone
+```
+
+## 3. Set up govuk-docker
 
 We use a Docker environment - [govuk-docker][] - for local development.
 
 👉 [Learn about how we use Docker](/manual/intro-to-docker.html)
 
 👉 [Get setup with govuk-docker][govuk-docker]
 
-## 3. Get SSH access to integration
+[govuk-docker]: https://github.com/alphagov/govuk-docker
 
-👉 [Get SSH access to integration](/manual/get-ssh-access.html)
+## 4. Get SSH access to integration
 
-## 4. Get AWS access
+### Get access
 
-👉 [Setup your AWS account](/manual/set-up-aws-account.html)
+Ask somebody with access to add your SSH username (`firstnamelastname`) to the [user monitoring system][user-reviewer].
+
+[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
+
+### Create a user to SSH into integration
+
+User accounts in our integration environments are managed in the [govuk-puppet][] repository.
+
+    mac$ mkdir ~/govuk
+    mac$ cd ~/govuk
+    mac$ git clone git@github.com:alphagov/govuk-puppet.git
+
+Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with your username and the public key you created when you set up your GitHub account above. Your file should use the `firstnamelastname.pp` format.
+
+```
+class users::johnsmith {
+  govuk_user { 'johnsmith':
+    fullname => 'John Smith',
+    email    => 'john.smith.123@digital.cabinet-office.gov.uk',
+    ssh_key  => 'this public key will be a few lines long (copy the output from `more ~/.ssh/alphagov.pub`). It should begin with `ssh-rsa AAA` and end with `== john.smith.123@digital.cabinet-office.gov.uk`. You may need to add the email address to the end of your public key manually.',
+  }
+}
+```
+
+Add the name of your manifest (your username) into the list of `users::usernames` in [`hieradata_aws/integration.yaml`][integration-aws-hiera] for integration and in [`hieradata/integration.yaml`][integration-hiera] for CI.
+
+Create a pull request with these changes. Once it has been [reviewed by a member of the GOV.UK team][merging], you can merge it and it will automatically deploy to the integration environment.
+
+[govuk-puppet]: https://github.com/alphagov/govuk-puppet
+[integration-aws-hiera]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/integration.yaml
+[integration-hiera]: https://github.com/alphagov/govuk-puppet/blob/master/hieradata/integration.yaml
+[merging]: /manual/merge-pr.html
+
+### Access remote environments
```

Comment:
I think add somewhere in here that it can take up to 30 minutes after the PR has been merged